### DISCUSSION TITLE
Optimizations after OS install

### DISCUSSION THREAD

**Discussion by mail2onur** [ChrisTitusTech/linutil#439] [Ideas]
How about implementing some basic system changes, optimizations to linutil post-OS install?

For example, there are a lot of tweaks at this web site. This is mainly for Ubuntu/Debian distros but the same can be done for Arch and Fedora based distros too by changing commands. These commands can be added to linutil and can be run all at once via scripts;

[https://easylinuxtipsproject.blogspot.com/p/1](https://easylinuxtipsproject.blogspot.com/p/1.html)

**Comment by koibtw** [comment]
Your hyprlink is broken.

**Reply by mail2onur** [reply]
Please copy-paste the address. Github forum link does not work for some reason

**Reply by koibtw** [reply]
I'll fix it for you

EDIT: done. see now
